scripts/give-ns-privileges-to-team.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+
+set -o errexit
+trap 'echo "Aborting due to errexit on line $LINENO. Exit code: $?" >&2' ERR
+set -o errtrace
+set -o pipefail
+
+if [ $# -ne 3 ] || [ "$#" == "--help" ] || [ "$#" == "-h" ] || ([ $1 != "admin" ] && [ $1 != "edit" ] && [ $1 != "view" ]); then
```

Comment:
@larsks you are right, if someone passing '--help' it will echo the usage because the number of parameters is not right and not because of the content of this parameter. Thank you I didn't notice it, I will use your suggestion for $1 and the layout.